accelerator/http.h

```diff
@@ -0,0 +1,53 @@
+#ifndef ACCELERATOR_HTTP_H_
+#define ACCELERATOR_HTTP_H_
+
+#include <microhttpd.h>
+#include <stdbool.h>
+#include "accelerator/apis.h"
+#include "accelerator/config.h"
+#include "accelerator/errors.h"
+#include "utils/logger_helper.h"
```

Comment:
I don't see any related usage. Do you plan to have any logging message here?

Reply:
Not for now, I'll remove it.